Sources/Remote Logging/Crash Logging/CrashLoggingDataProvider.swift

```diff
@@ -28,4 +29,41 @@ public extension CrashLoggingDataProvider {
     var shouldEnableAutomaticSessionTracking: Bool {
         return false
     }
+
+    var enableAutoPerformanceTracking: Bool {
+        switch performanceTracking {
+        case .enabled: return true
+        case .disabled: return false
+        }
+    }
+
+    var tracesSampleRate: Double {
+        guard case .enabled(let config) = performanceTracking else { return 0.0 }
+        return config.sampleRate
+    }
```

Comment:
I considered adding something like

```swift
extension PerformanceTracking {

  var configuration: PerformanceTracking.Configuration? { /* return value if .enabled, .none otherwise */ }
}
```

Then simplify these method like:

```swift
performanceTracking.configuration?.tracesSampleRate ?? 0.0
```

I decided against it because I feel like `guard case .enabled...` makes the code more explicit.